go.mod

```diff
@@ -0,0 +1,21 @@
+module sqlflow.org/sqlflow
+
+require (
+	github.com/apache/thrift v0.13.0 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
+	github.com/beltran/gohive v1.0.0 // indirect
```

Comment:
@typhoonzero How come such `//indirect` dependency entries? Did you manually add them? If so, why did you do that, instead of leaving these indirect dependencies using their default versions?

Reply:
`go.mod` and `go.sum` is generated automatically by running `go test`.